crowbar_framework/app/models/node_object.rb

```diff
@@ -1360,7 +1360,7 @@ def disk_release(device, owner)
     if device
       crowbar_wall[:claimed_disks] ||= {}
 
-      unless disk_owner(device) == owner
+      if owner.empty? || disk_owner(device) != owner
```

Comment:
Well, it is not _currently_ happening - not when `transition_to_readying` is called in `readying` state. At that point , there's no problem with empty owner, but indeed a problem with wrong disk data, so no disk is actually claimed for RAID.
It happens when I move the call of `process_raid_claims` to `ready` state... which again might indicate that such move is not a correct fix for the bug...